ETC Is a Commodity, ETH Is Security (#1089)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-02-28-ethereum-classic-is-a-commodity-ethereum-is-a-security/index.md

### 3. A reasonable expectation of profit: 

As said before, although both ETH and ETC may be used purely as units of exchange, they may also be used as a store of value for price appreciation, therefore both qualify as having a reasonable expectation of profit.

### 4. Derived from the efforts of others: 

For the same reasons that Ethereum is now a common enterprise it is also a system where the expectation of the success of the system, and thus its future profits, are derived from the efforts of others. 

These “others” will be perfectly identifiable, as they will be 4 or 5 large staking pools and the Ethereum Foundation, and all together will act as a single enterprise. 

The PoS system is not a free merit system as PoW but a totally collaborative partnership between the pools and the validators, and the validators are essentially the same as the pools because they are their submissive contractors.

In the system, validators divide their tasks on a per block basis where one produces a block, then sends it to the other validators who vote on the block, and then send it to the the rest of the network who must accept them without objection.

> Because ETC is PoW and decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in their complete isolation, send them to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 

Reviewer: Line 76: Rewrite for clarity
```suggestion
Because ETC uses proof of work and is, therefore, decentralized, miners may pop up or leave whenever they want from anywhere in the world, compete to build blocks in complete isolation, send blocks produced to the rest of the network for verification, and get paid based on their sole merit. No other filters or conditions exist. 
```

> In ETC, the same reasons why it was a commodity before remains today: There is no common enterprise, therefore the future value of the token is just determined by future general adoption by the market, but not from a defined and collaborative group of centralized pool operators or managers as in Ethereum.

Reviewer: Line 78:  Rewrite for clarity
```suggestion
For ETC, the same reasons why it was a deemed commodity in 2015 remain valid today: There is no common enterprise, therefore the future value of the token is solely dependent upon future general adoption by the market, not by the activities of a defined collaborative group of centralized operators or managers as is now the case with Ethereum.
```

## A Historical Perspective of ETC’s and ETH’s Legal Status

![The crowdsale.](./2.png)

From 2014 to 2016 both Ethereum and Ethereum Classic were one project, so we refer to them in this period as ETH/ETC. 

> The project was financed with a crowdsale. Based on the Howey test it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 

Reviewer: Line 86: Add comma
```suggestion
The project was financed with a crowdsale. Based on the Howey test, it may be inferred that the ETH/ETC [crowdsale was, indeed, a security](https://twitter.com/WendellMoone/status/1623760982672564226). 
```

It was [founded](https://en.wikipedia.org/wiki/Ethereum) by Vitalik Buterin; then he partnered with several individuals to form a group of co-founders; they promised the expectation of profits out of the investment; created a foundation with the name of the network, even registering its trademarks; sold a unit of value previous to the creation of the cryptocurrency to gather funds for development; and promoted the crowdsale and Ethereum’s features, benefits, and appreciation potential just like any initial public offering of a stock would have been promoted.

However, this status as a security was likely limited to the period between the crowdsale and the launch of the network on July 30th of 2015. 

![ETH/ETC launch.](./3.png)

On July 30th of 2015, the initial securities of the project were transformed into the ETH cryptocurrency, which was subsequently categorized as a commodity by the CFTC.

This step may be interpreted to be an exchange and exit of the initial common enterprise between investors, Vitalik Buterin, his partners, and the Ethereum Foundation as the director of the project.

> From here, the project became a truly decentralized, proof of work based, public blockchain. 

Reviewer: Line 98: Usage
```suggestion
From there, the project became a truly decentralized, proof of work based, public blockchain. 
```

![The Ethereum split.](./7.png)

> Even when Ethereum split from the ETC mainnet in 2016 due to the DAO hard fork, ETH kept its decentralization because it remained as a proof of work blockchain.

Reviewer: Line 102: Usage; Capitalization
```suggestion
Even after Ethereum split from the ETC mainnet in 2016 due to the TheDAO hard fork, ETH kept its decentralization because it remained a proof of work blockchain.
```

This period lasted until September 15th of 2022.

![The ETH migration to Proof of Stake.](./8.png)

> Starting in September 15th of 2022, Ethereum cannot be described as a decentralized project because it migrated to Proof of Stake. Indeed, the moment it migrated, 51% of the blocks were censored by mega-staking pools to comply with international sanctions imposed by the U.S. Office of Foreign Assets Control (OFAC). And, a few months later up to 70% of its blocks were censored.

Reviewer: Line 108: Remove preposition; Remove conjunction
```suggestion
Beginning September 15th of 2022, Ethereum cannot be described as a decentralized project because it migrated to Proof of Stake. Indeed, the moment it migrated, 51% of the blocks were censored by mega-staking pools to comply with international sanctions imposed by the U.S. Office of Foreign Assets Control (OFAC). A few months later, up to 70% of its blocks were censored.
```

As said before, PoS depends on large and entrenched pool operators that control the network. This, together with the control and influence that the Ethereum Foundation had from before, and the fact that the developers have incredible clout in the decisions about the protocol (for example, changing the supply of the token 6 times in its history) make it a centralized project dependent on others from the units’ of value perspective.

![BTC remains a commodity and ETC as well.](./6.png)

> In the meantime, Bitcoin has always maintained its status as a commodity as determined by the CFTC in 2015, and ETC is functionally a commodity, just like BTC, because it has exactly the same consensus design with its decentralist guarantees.

Reviewer: Line 114:  Remove run-on sentence
```suggestion
In the meantime, Bitcoin has always maintained its status as a commodity as determined by the CFTC in 2015. ETC is functionally a commodity, just like BTC, because it has exactly the same consensus design with its decentralist guarantees.
```

However, ETC as the base layer of these technologies is still a generic commodity.

The proof of stake algorithm in Ethereum implies a constant collaboration between validators, who are contractors of the pool operators. They produce blocks, vote on these blocks, and then send them to the rest of the network who have to accept them without question.

Additionally, participation by stakers from the public or node operators to run validating clients is filtered and restricted by pool operators. There is no decentralized free entry and exit from the system, therefore it is clearly a common enterprise.

ETH/ETC went through a centralized sale process when it was promoted during the crowdfunding, but then it became a decentralized blockchain when it was launched and was a proof of work system. 

When ETH split from the ETC mainnet in 2016, ETC was separated from the main community of developers and leaders of the Ethereum Foundation and became even more decentralized at the social layer. This is when its Code Is Law principle was established.

ETC never had a defined group of promoters. In fact, all the different groups of constituents in ETC rotate and migrate constantly.

> For all the reasons stated above, we conclude that Ethereum Classic remains a commodity and Ethereum is a security.

Reviewer: Line 134: Rewrite for accuracy
```suggestion
For all the reasons stated above, we conclude that Ethereum Classic remains a commodity while Ethereum has reverted to being a security.
```
